### DISCUSSION TITLE
Migrate a repo from --encryption=repokey to --encryption=keyfile

### DISCUSSION THREAD

**Discussion by vbc4ejyg** [borgbackup/borg#6200] [Q&A] [answered]
I created a remote backup that took some time to transfer, but now wishing I would have stored the key locally instead of remote.  Is it possible to manually switch this without disturbing the repo?  

Something changing key-type from 3 to 0?  and creating the keyfile locally in .config/borg/keys?

Seems like there has to be a way, but I haven't figured it out yet.

**Comment by ThomasWaldmann** [comment, marked as answer]
Not easily possible, see there: https://github.com/borgbackup/borg/issues/510
